assets/javascripts/discourse/components/translate-topic-title.js

```diff
@@ -0,0 +1,32 @@
+import Component from "@ember/component";
+import { ajax } from "discourse/lib/ajax";
+
+export default Component.extend({
+  tagName: "span",
+
+  actions: {
+    translate() {
+      const topic = this.topic;
+      const originalTitle = topic.title;
+
+      this.set("translating", true);
+
+      return ajax("/translator/translate", {
+        type: "POST",
+        data: { topic_id: topic.id },
+      })
+        .then(function (res) {
+          topic.setProperties({
+            title: res.translation,
+            fancy_title: res.translation,
```

Comment:
This makes me a little nervous because if there is a bug somewhere, a user could inadvertently edit a topic's title into the translated language. Is there a way we can do this without messing with the model's attributes?